make-it/Makefile

```diff
@@ -176,8 +176,7 @@ lARMv5:
 
 #========= Use makefile designed for emscripten builds
 vjs: $(OUTPUT_DIR)
-	# Just buid everything
-	make -f EmMakefile vjs
+	emmake make -f EmMakefile vjs
```

Comment:
Why is emmake necessary?  This will fail on Mac, because emmake contains a #!/usr/bin/env python2, but the system python2 is just called python; and the version-mandled names have the minor version too, e.g. python2.7.